defaults/main.yml

```diff
@@ -12,3 +12,13 @@ logging_conf: {"files":{"rotateeverybytes":10485760}}
 output_conf: {"elasticsearch":{"hosts":["localhost:9200"]}}
 beats_pid_dir: "/var/run"
 beats_conf_dir: "/etc/{{beat}}"
+beats_data_dir: "/var/lib/{{beat}}"
+beats_home_dir: "/usr/share/{{beat}}"
+enable_keystore: false
+es_output_pass: ''
+es_output_ssl_ca: ''
+es_mon_ssl_ca: ''
+
+es_enable_sll: false
```

Comment:
Typo here on SSL, should be `es_enable_ssl`

Reply:
I will correct that.  This typo appears on multiple places.

Reply:
Typo correction SLL->SSL
15a858c